src/App.jsx

```diff
@@ -16,7 +16,7 @@ function App() {
         <Grid item xs={3}>
```

Comment:
I would change this to `xs={2}` and its sibling `xs={10}`, then remove the `width: 'fit-content'` style from the `Navigation` component. Right now there is too much of a margin between the sidebar and the form content.

Reply:
Damn. that looks way better.